ucx.spec

```diff
@@ -19,14 +19,21 @@
 %bcond_with    xpmem
 %bcond_with    vfs
 
+%global major 1
+%global minor 13
+%global bugrelease 0
+%global prerelease rc1
+
+%global dl_version %{major}.%{minor}.%{bugrelease}
+
```

Comment:
Awesome that you ported this standard from our other packaging projects into this one.